InnerEye/Scripts/submit_for_inference.py

```diff
@@ -162,12 +162,14 @@ def submit_for_inference(args: SubmitForInferenceConfig) -> Optional[Path]:
     return download_path
 
 
-def main(args: Optional[List[str]] = None) -> None:
+def main(args: Optional[List[str]] = None, project_root: Optional[Path] = None) -> None:
     """
     Main function.
     """
     logging_to_stdout()
-    submit_for_inference(SubmitForInferenceConfig.parse_args(args))
+    inference_config = SubmitForInferenceConfig.parse_args(args)
+    azure_config = AzureConfig.from_yaml(inference_config.settings, project_root=project_root)
```

Comment:
For consistency in the command line interface, could we allow users to override the AzureConfig parameters via the command line, the way we do in runner.py?

The same goes for report_structure_extremes.py and tensorboard_monitor.py.

Reply:
If we add the full azure_config, it means that --help will no longer be really helpful, because that will bloat with 20 parameters that are not really relevant. I've added a --cluster argument though, as discussed over IM.